spec/helpers/feedbacks_helper_spec.rb

```diff
@@ -29,6 +29,17 @@ class Klass
         klass.open_requests(feedback.user).should == 1
       end
     end
+
+    describe 'invitations and submitted feedback' do
+      let(:user) { create(:user) }
+      let!(:invitations) { create_list(:invitation, 5, email: user.email) }
+      let!(:feedback) { create(:feedback, user: user) }
+
+      it 'should return the count of invitations and feedback' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.